skins/moono-lexicon/icons/icons.json

```diff
@@ -4,36 +4,59 @@
         "align-image-center": "align-image-center",
         "align-image-left": "align-image-left",
         "align-image-right": "align-image-right",
-        "anchor": "flag-full",
+        "anchor": {
+            "ltr": "flag-full"
+        },
         "blockquote": "quote-right",
         "bold": "bold",
-        "bulletedlist": "list-ul",
+        "bulletedlist": {
+            "ltr": "list-ul"
+        },
         "copy": "paste",
         "cut": "cut",
-        "find": "search",
+        "find": {
+            "ltr": "search",
+            "rtl": "search"
```

Comment:
Is this mean to be a different `search`? Or if they're the same, why separate `rtl` and `ltr`?

Reply:
Same, CKEditor have it as directional so we need to specify the icon for both directions. Probably we'll need to specify other icon, something like `search-rtl`